### ISSUE TITLE
Linter proposal: allow only imports at top of file

### ISSUE THREAD

**Issue by lencioni** [brigade/scss-lint#732]
Bad:

```scss
.foo {
  color: red;
}

@import 'something';
```

Bad:

```scss
.foo {
  @import 'something';
}
```

Good:

```scss
// Comments are okay
@import 'something';

.foo {
  color: red;
}
```

**Comment by tysongach** [comment]
:+1:
